### PR TITLE
Feature/8979 blog link info screen for statistic data

### PR DESCRIPTION
**Pull request by ArturFriesen**
## Description
Adds a text and link to statistic cards info screen.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8979

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.links.strings [hunk 1]
```diff
@@ -41,3 +41,5 @@
 "Test_Centers_FAQ_Link" = "https://www.coronawarn.app/de/faq/#where_can_i_get_tested";
 
 "HealthCertificate_InvalidSignature_FAQLink" = "https://www.coronawarn.app/de/faq/#hc_signature_invalid";
+
+"Statistics_Info_Blog_Link" = "https://www.coronawarn.app/en/blog/2021-08-05-statistictiles";
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.links.strings:45]: en in de?

Comment (NataliaLemmerth) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.links.strings:45]: Please change it to "https://www.coronawarn.app/de/blog/2021-08-05-statistictiles"

Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.links.strings]: Changed en link to de for de.
Added en link for all others.

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**